task1/index.html

```diff
@@ -0,0 +1,230 @@
+<!DOCTYPE html>
+<!-- Объявление формата документа -->
```

Comment:
try to avoid comments in html/css/js. Comments are acceptable only if there are really needed, like dirty hack explanation, complex functionality. Code should be self described.